starter/script.js

```diff
@@ -1 +1,93 @@
-// put code here!
+var side = prompt ("Your shirt is atrocious! I'm taking your dog! You'll have to solve my labyrinth. You have 13 mins. Pick your path! Left or Right! ");
+
+if (side == "left") {
+  alert("Boog of Eternal Strentch");
+
+  var boog_direction = prompt ("Oh! I see you're in the Boog of Eternal Stentch. Which way do you want to go. Left, or right. Your choice.");
+  var side;
+      if (side == "left") {
```

Comment:
Compare this prompt and conditional to the first prompt and corresponding conditional.  What is different?